docs/api-docs/getting-started/api-rate-limits.mdx

```diff
@@ -0,0 +1,97 @@
+# Adaptive concurrency limits
+
+BigCommerce aims to provide outstanding platform reliability. To maintain dependability, adaptive concurrency limits are applied to both REST and GraphQL APIs. These limits enhance quality for merchants by regulating multiple API calls. In this doc we'll outline how these limits work on the BigCommerce platform.
```

Comment:
Are we using "adaptive concurrency limits" as a replacement to rate limits? Not sure where to add the nomenclature if nots not intended to replace that.

Reply:
we should stick to rate limiting, it is better to not mention words "adaptive" etc, these are internal details that are a subject to change